compiler/rustc_target/src/spec/wasm32_unknown_emscripten.rs

```diff
@@ -17,10 +17,6 @@ pub fn target() -> Target {
     post_link_args.insert(
         LinkerFlavor::Em,
         vec![
-            "-s".into(),
-            "ERROR_ON_UNDEFINED_SYMBOLS=1".into(),
-            "-s".into(),
-            "ASSERTIONS=1".into(),
             "-s".into(),
             "ABORTING_MALLOC=0".into(),
```

Comment:
Why you are there you could also merge these two lines into a single `-sABORTING_MALLOC=0".into(),`?